wandb/sdk/wandb_settings.py

```diff
@@ -50,7 +50,176 @@ def _path_convert(*args: str) -> str:
+        disable_git (bool): Whether to disable capturing the git state.
+        disable_job_creation (bool): Whether to disable the creation of a job artifact for W&B Launch.
+        docker (str | None): The Docker image used to execute the script.
+        email (str | None): The email address of the user.
+        entity (str | None): The W&B entity, like a user or a team.
+        force (bool): If true, the `force` flag will be passed to wandb.login().
+        fork_from (RunMoment | None): Specifies a run to fork from.
+        git_commit (str | None): The git commit hash to associate with the run.
+        git_remote (str): The git remote to use (defaults to "origin").
+        git_remote_url (str | None): The URL of the git remote repository.
+        git_root (str | None): The root directory of the git repository.
+        heartbeat_seconds (int): Interval in seconds between heartbeat signals sent to the W&B servers.
+        host (str | None): The hostname of the machine running the script.
+        http_proxy (str | None): The custom proxy servers for http requests to W&B.
+        https_proxy (str | None): The custom proxy servers for https requests to W&B.
+        identity_token_file (str | None): Path to file containing an identity token (JWT) for authentication.
+        ignore_globs (tuple[str, ...]): Unix glob patterns relative to `files_dir` to not upload.
+        init_timeout (float): Time in seconds to wait for the wandb.init call to complete before timing out.
+        insecure_disable_ssl (bool): Whether to insecurely disable SSL verification.
+        job_name (str | None): Name of the job running the script.
+        job_source (Literal["repo", "artifact", "image"] | None): Source type for Launch.
+        label_disable (bool): Whether to disable automatic labeling features.
+        launch (bool): Flag to indicate if the run is being launched through W&B Launch.
+        launch_config_path (str | None): Path to the launch configuration file.
+        login_timeout (float | None): Time in seconds to wait for login operations before timing out.
+        mode (Literal["online", "offline", "dryrun", "disabled", "run", "shared"]): The operating mode for W&B logging and synchronization.
+        notebook_name (str | None): Name of the notebook if running in a Jupyter-like environment.
+        program (str | None): Path to the script that created the run, if available.
+        program_abspath (str | None): The absolute path from the root repository directory to the script that created the run.
+          Root repository directory is defined as the directory containing the .git directory, if it exists. Otherwise, it's the current working directory.
+        program_relpath (str | None): The relative path to the script that created the run.
+        project (str | None): The W&B project ID.
+        quiet (bool): Flag to suppress non-essential output.
+        reinit (bool): Flag to allow reinitialization of a run.
```

Comment:
```suggestion
        reinit (bool): Flag to allow reinitialization of a run.  If not set, when an active run exists, calling `wandb.init()` returns the existing run instead of creating a new one.
```